caffe2/operators/conv_gradient_op.cc

```diff
@@ -4,8 +4,85 @@
 
 namespace caffe2 {
 
+std::vector<TensorShape> TensorInferenceForConvGradient(
+    const OperatorDef& def,
+    const std::vector<TensorShape>& in) {
+  CAFFE_ENFORCE(in.size() >= 2, "ConvGradient expects at least 2 inputs");
```

Comment:
This should be 3 same as CostInferenceForConvGradient?
``
  CAFFE_ENFORCE_EQ(in.size(), 3, "ConvGradient expects 3 inputs");
``